### ISSUE TITLE
Potential security issue of using password given for first user as root password

### ISSUE THREAD

**Issue by Mikaela** [Antergos/Cnchi#602]
*I guess I can clear the issue template in this case.*

At IRC someone asked something which made MichaelTunnell say that they think that Antergos doesn't enable root account by default. I said that I think it uses the password of first user created in Cnchi and confirmed with @lots0logs that it's so.

I then wondered  that *It might be security hole if the password is leaked and no one remembers to change it* **and** *Is it mentioned in Cnchi or anywhere?* where @lots0logs replied *I'm not sure. I cant remember I mean. Open an issue on github and we'll check it 😉*

**Comment by ReemZ-nl** [comment]
Nope, not mentioned in Cnchi, and yes, definitely a security hole.

**Comment by karasu** [comment]
The user you create is an "admin" user as it can sudo. Moreover, we do not disable root account and we set it the very same exact password. What do you suggest? Disable root account? Ask for a different root password?

**Comment by ReemZ-nl** [comment]
I suggest setting a different root password. The account does come in handy (I'm a 'su -' person myself, not much of a 'sudo'er) so I see no reason to disable it completely (but don't have root able to log into the graphical environment!) but asking for different passwords leaves less of a security problem should the user password get compromised for any reason. Basically the same reason why you'd use different passwords on different websites.

**Comment by lots0logs** [comment]
Yes, I agree that using different passwords would technically be more secure. However, that doesn't mean the way we set it up now is insecure or wrong. At the end of the day, this is simply a matter of preference. For the record, I prefer how it is now, and I'm against making the change proposed here. There is a fine line between helpful preconfiguration and forcing one's own ideology onto users of the software. The change proposed here crosses that line IMO.

**Comment by ReemZ-nl** [comment]
Couldn't the installer at least offer the option, like for example Calamares does, have the user tick or untick a box? IMO that's the only way to not cross the line mentioned.

**Comment by Mikaela** [comment]
>  What do you suggest? Disable root account? Ask for a different root password?

Why not disable root account?

**Comment by ReemZ-nl** [comment]
Because the root account does have its purpose, for example in recovery mode.

**Comment by lots0logs** [comment]
> Couldn't the installer at least offer the option, like for example Calamares does, have the user tick or untick a box? IMO that's the only way to not cross the line mentioned.

That's reasonable. I'm okay with making it an option though not enabled by default. :+1:
